src/sentry/static/sentry/app/components/search/sources/routeSource.jsx

```diff
@@ -0,0 +1,90 @@
+import {flattenDepth} from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {createFuzzySearch} from '../../../utils/createFuzzySearch';
+import accountSettingsNavigation from '../../../views/settings/account/navigationConfiguration';
+import organizationSettingsNavigation from '../../../views/settings/organization/navigationConfiguration';
+import projectSettingsNavigation from '../../../views/settings/project/navigationConfiguration';
+import replaceRouterParams from '../../../utils/replaceRouterParams';
+
+const navigationItems = flattenDepth(
+  [
+    accountSettingsNavigation,
+    projectSettingsNavigation,
+    organizationSettingsNavigation,
+  ].map(config =>
```

Comment:
maybe do this on two lines or define a named helper method or make some other change make it easier to read/extend? it's pretty dense

Reply:
I think you'd like clojure though :P